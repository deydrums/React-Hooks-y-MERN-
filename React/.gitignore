# proyectos
/00-proyectos