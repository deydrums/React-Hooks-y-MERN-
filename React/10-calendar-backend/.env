PORT=80
DB_CNN=mongodb+srv://deydrums:7VaOymNqlPNdUn12@cluster0.id2bg.mongodb.net/CalendarApp
SECRET_JWT_SEED=Esto-Es-una-PalAbRa-S3kr2t4-j3j3j3