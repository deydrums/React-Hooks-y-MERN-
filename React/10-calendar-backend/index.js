const express = require('express');
require('dotenv').config();
//Ejecutar express
var app = express();

//Directorio publico
app.use(express.static('public'));

//Rutas
//Reescribir rutas
app.use('/api/auth', require('./routes/auth'));

//TODO: CRUD: Eventos

//Escuchar peticiones
app.listen(process.env.PORT,()=>{
    console.log(`Servidor corriendo en puerto ${process.env.PORT}`)
});