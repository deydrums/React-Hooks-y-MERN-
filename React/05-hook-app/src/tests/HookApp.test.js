import { shallow } from "enzyme";
import React from 'react';
import { HookApp } from "../HookApp";


describe('Pruebas en el componente <HookApp></HookApp>', () => {

    test('Debe de mostrarse el componente <HookApp></HookApp>', () => {
        const wrapper = shallow(<HookApp/>);
        expect(wrapper).toMatchSnapshot();
    })
})