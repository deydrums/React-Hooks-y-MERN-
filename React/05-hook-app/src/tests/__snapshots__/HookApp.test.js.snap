// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HookApp></HookApp> Debe de mostrarse el componente <HookApp></HookApp> 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
